tracer/src/Datadog.Trace/Configuration/TracerSettings.cs

```diff
@@ -492,6 +500,17 @@ internal TracerSettings(IConfigurationSource? source, IConfigurationTelemetry te
         [ConfigKey(ConfigurationKeys.TraceEnabled)]
         internal bool TraceEnabledInternal { get; private set; }
 
+        /// <summary>
+        /// Gets a value indicating whether Appsec standalone is enabled.
+        /// Default is <c>false</c>.
+        /// </summary>
+        /// <seealso cref="ConfigurationKeys.ExperimentalAppsecStandaloneEnabled"/>
+        [GeneratePublicApi(
+            PublicApiUsage.TracerSettings_ExperimentalAppsecStandaloneEnabled_Get,
+            PublicApiUsage.TracerSettings_ExperimentalAppsecStandaloneEnabled_Set)]
```

Comment:
I _think_ going forwards we're don't want any new public setters for config. That ones that exist are older config that is already public so we can't remove them easily. IIUC the goal is to encourage users to use other config methods instead of code, because it requires adding the nuget package, which can cause issues when mixing with automatic instrumentation.

cc @andrewlock

Reply:
On second thought, since this setting is designed to be temporary, maybe we should have a public property at all (not even the getter), so we don't break users when we remove it later.

Reply:
Okay I see 😊, I removed all reference to the public API in 72120d1fb9ce61afb49361747c6888665a04293e